packages/sync/legacy/class.jetpack-sync-main.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * This class hooks the main sync actions.
+ *
+ * @package jetpack-sync
+ */
+
+/**
+ * Jetpack Sync main class.
+ */
+class Jetpack_Sync_Main {
+	/**
+	 * Initialize the main sync actions.
+	 */
+	public static function init() {
+		// Check for WooCommerce support.
+		add_action( 'plugins_loaded', array( 'Jetpack_Sync_Actions', 'initialize_woocommerce' ), 5 );
```

Comment:
```suggestion
		add_action(
			'plugins_loaded',
			array( 'Jetpack_Sync_Actions', 'initialize_woocommerce' ),
			5
		);
```

Reply:
Thanks, but I disagree and prefer the original one as it's more readable and WP-like to me.